README.md

```diff
@@ -230,14 +307,13 @@ RNCallKeep.hasDefaultPhoneAccount(options);
 
 ### didReceiveStartCallAction
 
-User start call action from _Recents_ (Or _Contact_ on Android) in built-in phone app.
+Device sends this event once it decides the app is allowed to start a call, either from the built-in phone screens (iOS/_Recents_, Android/_Contact_),
+or by the app calling `RNCallKeep.startCall`.
 
-Try to start your call action from here (e.g. get credentials of the user by `data.handle` and/or send INVITE to your SIP server)
-
-After all works are done, remember to call `RNCallKeep.startCall(uuid, calleeNumber)`
+Try to start your app call action from here (e.g. get credentials of the user by `data.handle` and/or send INVITE to your SIP server)
 
 ```js
-RNCallKeep.addEventListener('didReceiveStartCallAction', ({ handle }) => {
+RNCallKeep.addEventListener('didReceiveStartCallAction', ({ number, callUUID, name }) => {
```

Comment:
The argument here is still called `handle` instead of `number`.

Reply:
number instead of handle*

Reply:
I mean [this argument](https://github.com/sangoma/react-native-callkeep/blob/master/android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java#L527) or `RNCallKeepDidReceiveStartCallAction` is still called `handle`. But [this one](https://github.com/react-native-webrtc/react-native-callkeep/pull/64/files#diff-80e75216e5090135007ce3377e5d957bR498) is called `number`.

Reply:
Ah Sangoma have fixed this their side (I brought it up with them). Not sure if Kyle has pushed it in yet.They've changed  Android everything back to handle I think....

Reply:
Think this stems from the iOS DidReceiveStartCallAction sending `handle` when calling from Recents and sending `number` when it's from within the app. Will change to be consistent

Reply:
I merged in your README.md with our update after this conversation. Should be resolved now.